Modules/Filtering/DiffusionTensorImage/include/itkDiffusionTensor3DReconstructionImageFilter.h

```diff
@@ -251,7 +251,7 @@ class ITK_TEMPLATE_EXPORT DiffusionTensor3DReconstructionImageFilter
     {
       itkExceptionMacro(<< "Gradient direction " << idx << "does not exist");
     }
-    return m_GradientDirectionContainer->ElementAt(idx + 1);
+    return m_GradientDirectionContainer->ElementAt(idx);
```

Comment:
Wondering it the Slicer3D folks use this diffusion class at all, but this might be interesting in case they do.